src/main/java/seedu/address/logic/commands/CloneCommand.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Birthdate;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Gender;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Race;
+import seedu.address.model.person.Religion;
+import seedu.address.model.person.Survey;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Clones an existing person in the address book.
+ */
+public class CloneCommand extends Command {
+
+    public static final String COMMAND_WORD = "clone";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Clones the details of the person identified "
+            + "by the index number used in the displayed person list. "
+            + "A new person with different name but same details will be added to the address book.\n"
+            + "Parameters: INDEX (must be a positive integer) \n"
```

Comment:
Maybe say that the index of the person needs to exist and not just a positive integer.